src/libraries/Native/Unix/System.Globalization.Native/pal_icushim_internal.h

```diff
@@ -46,17 +46,21 @@
 
 #include "icu.h"
 
-#ifndef __typeof
-#define __typeof decltype
-#endif
-
 #define HAVE_SET_MAX_VARIABLE 1
 #define UDAT_STANDALONE_SHORTER_WEEKDAYS 1
 
 #endif
 
 #include "pal_compiler.h"
 
+#ifndef TYPEOF
```

Comment:
We are using `__typeof` in other shims too. It may be nice to move this to `pal_compiler.h` for consistency.

Reply:
Sounds good, I'll do that cleanup.

Reply:
Done, thanks.